WindowsMSys2_InstallGuide.md

```diff
@@ -1,88 +1,73 @@
-
-Restart console  
-
-       pacman -Su  
-
-Install all default programms  
-
-       pacman -S base-devel git mercurial cvs wget p7zip gcc perl ruby python2  
+Download MSYS2 from [MSYS2](http://www.msys2.org/). Get the i686 version
+if you run a 32-bit Windows or the x86_64 if you run a 64-bit Windows.
 
-Choose only w64-i686 if you want only compilation in 32b architecture.  
+Run the installer and install MSYS2.
 
-       pacman -S mingw-w64-i686-toolchain mingw-w64-x86_64-toolchain  
+At the end of the installation, it'll automatically open an MSYS shell terminal.
+You can also find it on the start menu as MSYS2 MSYS. This is the shell you'll 
+use to install packages with pacman and do maintenance but NOT to build
+RetroShare.
 
-### Install other binutils:   
-       pacman -S mingw-w64-i686-miniupnpc mingw-w64-x86_64-miniupnpc  
-       pacman -S mingw-w64-i686-sqlite3 mingw-w64-x86_64-sqlite3  
-       pacman -S mingw-w64-i686-speex mingw-w64-x86_64-speex  
-       pacman -S mingw-w64-i686-speexdsp mingw-w64-x86_64-speexdsp
-       pacman -S mingw-w64-i686-opencv mingw-w64-x86_64-opencv
-       pacman -S mingw-w64-i686-ffmpeg mingw-w64-x86_64-ffmpeg  
-       pacman -S mingw-w64-i686-libmicrohttpd mingw-w64-x86_64-libmicrohttpd  
-       pacman -S mingw-w64-i686-libxslt mingw-w64-x86_64-libxslt  
+First, update your MSYS2 environment to the latest version by typing:
 
-Add MSYS2 to PATH environment variable depends your windows  
+	pacman -Syu
 
-       ;C:\msys64\mingw32\bin
-       ;C:\msys32\mingw32\bin
+Close the terminal window.
 
+Run MSYS2 MSYS again and finish updating with:
 
-### Git Installation
+	pacman -Su
 
-Install Git Gui or other client: [Git Scm](https://git-scm.com/download/win)
+Install the default programs needed to build:
 
-Create a new directory named:  
+	pacman -S base-devel git wget p7zip gcc perl ruby python2
 
-       C:\Development\GIT  
+Install the 32-bit toolchain:
 
-Right-click on it and choose: *Git Bash Here*  
+	pacman -S mingw-w64-i686-toolchain
 
-Paste this text on git console:  
-git clone https://github.com/RetroShare/RetroShare.git  
+Install all needed dependencies:
 
+	pacman -S mingw-w64-i686-miniupnpc
+	pacman -S mingw-w64-i686-libmicrohttpd
+	pacman -S mingw-w64-i686-libxslt
+	pacman -S mingw-w64-i686-xapian-core
+	pacman -S mingw-w64-i686-sqlcipher
+	pacman -S mingw-w64-i686-qt5
 
-### Last Settings
+We're done installing MSYS2, close the shell terminal.
 
+### BUILDING RETROSHARE
 
-In QtCreator Option Git add its path: *C:\Program Files\Git\bin* 
-and select "Pull" with "Rebase"  
+Now run the MSYS2 MinGW 32-bit shell terminal (it's in the start menu).
+We will use it to checkout Retroshare and build it:
 
+	git clone https://github.com/RetroShare/RetroShare.git
 
-Open an MSys2 32|64 shell  
-Move to build_scripts:  
+Go to the RetroShare directory and configure to your liking, for example:
+	
+	cd RetroShare
+	qmake -r -Wall -spec win32-g++ "CONFIG+=debug" "DEFINES+=RS_AUTOLOGIN"
```

Comment:
You should change `"DEFINES+=RS_AUTOLOGIN"` into `"CONFIG+=rs_autologin"` as you can see into retroshare.pri file